content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -1790,6 +1790,27 @@ uses this annotation.
 See [annotations](https://kubernetes-sigs.github.io/aws-load-balancer-controller/latest/guide/service/annotations/)
 in the AWS load balancer controller documentation.
 
+### service.beta.kubernetes.io/aws-load-balancer-security-groups (deprecated) {#service-beta-kubernetes-io-aws-load-balancer-security-groups}
+
+Example: `service.beta.kubernetes.io/aws-load-balancer-security-groups: "sg-53fae93f,sg-8725gr62r"`
+
+Used on: Service
+
+The AWS load balancer controller uses this annotation to specify a comma seperated list
+of frontend securityGroups you want to attach to an NLB. Both name and ID of securityGroups
+are supported where name matches a `Name` tag, not the `groupName` attribute.
+
+When this annotation is added to a Service, the load-balancer controller attaches the security groups
+referenced by the annotation to the loadbalancer. This effectively allows your security team to manage the
+security groups assigned to the load balancer. On the other hand, when the annotation
+is omitted, the controller automatically creates a security group and attaches it to the load balancer.
```

Comment:
```suggestion
When this annotation is added to a Service, the load-balancer controller attaches the security groups
referenced by the annotation to the load balancer. If you omit this annotation, the AWS load balancer
controller automatically creates a new security group and attaches it to the load balancer.
```

Reply:
This looks better, i can add this in new commit.